jena-arq/src/main/java/org/apache/jena/sparql/engine/http/QueryEngineHTTP.java

```diff
@@ -532,6 +535,18 @@ else if ( actualContentType.equals(WebContent.contentTypeJSON))
         }
     }
 
+    @Override
+    public JsonArray execJson()
+    {
+        throw new NotImplementedException("JSON queries not implemented for remote calls") ;
```

Comment:
Minor: for better or worse, we have `org.apache.jena.atlas.lib.NotImplemented`.

Reply:
Coincidentally, I noticed this [lang] class here too yesterday, and was going to replace by that Atlas class... but first I searched other occurrences, and I think somewhere in a sparql module it was being used too. So I thought it was OK. Happy to replace this one by the atlas module class. Consider it done in some hours.